app/models/operatingsystem.rb

```diff
@@ -12,7 +12,7 @@ class Operatingsystem < ActiveRecord::Base
       :architectures, :architecture_ids, :architecture_names,
       :medium_ids, :medium_names,
       :os_default_templates_attributes, :os_parameters_attributes,
-      :ptable_ids, :ptable_names
+      :ptable_ids, :ptable_names, :provisioning_templates, :provisioning_template_ids
```

Comment:
all _ids attributes should have the corresponding _names too.

I think the line should be moved up too, it looks like these associations are in alphabetical order and roughly one association per line.